service/src/main/resources/application.yml

```diff
@@ -1,5 +1,6 @@
 logging:
   level.bio.terra.cbas: ${BATCHANALYSIS_LOG_LEVEL:info}
+  pattern.level: '%X{requestId} %5p'
```

Comment:
Is there a reason this was added back in?

Reply:
Accidental from merging with main. I'll remove this again.